features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/ServiceCatalogComponent.jsx

```diff
@@ -18,22 +18,29 @@
 
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import Listing from './Listing/Listing';
+import Listing from 'AppComponents/ServiceCatalog/Listing/Listing';
+import Overview from 'AppComponents/ServiceCatalog/Listing/Overview';
 
 /**
  * Have used key={Date.now()} for `Route` element in `/service-catalog`
  */
-const ServiceCatalog = () => {
+const ServiceCatalogComponent = () => {
     return (
         <Switch>
             <Route
                 exact
                 path='/service-catalog'
                 key={Date.now()}
-                component={Listing}
+                component={(props) => <Listing {...props} />}
+            />
+            <Route
+                path='/service-catalog/:service_uuid/overview'
+                key='/service-catalog/:service_uuid/overview'
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                component={(props) => <Overview {...props} />}
             />
         </Switch>
     );
 };
 
-export default ServiceCatalog;
+export default ServiceCatalogComponent;
```

Comment:
`ServiceCatalogComponent ` is a react component so why we need to append `Component` to the name 😃